### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by Anthariel**
This version adding some translations for the french community.

### CODE REVIEW

File: addons/medical/stringtable.xml [hunk 1]
```diff
@@ -665,126 +700,151 @@
...
       </Key>
       <Key ID="STR_ACE_MEDICAL_CHECK_PULSE_OUTPUT_1">
         <English>You find a Heart Rate of %2</English>
+        <French>Vous avez trouvé une fréquence cardiaque de %2</French>
         <Russian>Пульс %2 уд./мин.</Russian>
         <Spanish>El Pulso es %2</Spanish>
       </Key>
       <Key ID="STR_ACE_MEDICAL_CHECK_PULSE_OUTPUT_2">
-        <English>You find a weak Heart Rate</English>
+        <English>You find a faible fréquence cardiaque</English>
```
Comment (KoffeinFlummi) [inline addons/medical/stringtable.xml]: This seems wrong...

**Comment by KoffeinFlummi** [comment]
Thanks for your contribution! Please fix the line I commented on, then this is ready for merging.

**Comment by Anthariel** [comment]
Thanks KoffeinFlummi, It's now fixed.

Sorry for that.

**Comment by Anthariel** [comment]
Merci une fois de plus :+1: 

/ 523417b -> Is already here.

**Comment by Anthariel** [comment]
@KoffeinFlummi : (or anyone from ACE-Team), can I add translations for modules? (By default, they are not translated).

Thanks.

**Comment by KoffeinFlummi** [comment]
Thanks!

> @KoffeinFlummi : (or anyone from ACE-Team), can I add translations for modules? (By default, they are not translated).

> Thanks.

You would have to change the configs to look up the strings in the stringtable, feel free to create a pull request to do that.

**Pull request merged**